shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -753,6 +753,11 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
 
         result.setSelected(semanticsNode.hasFlag(Flag.IS_SELECTED));
 
+        // Heading support
+        if (Build.VERSION.SDK_INT >= 28) {
```

Comment:
I believe we usually compare this against the constants from Build.VERSION_CODES instead of hard-coding them.

Reply:
Good to know. I saw it both ways in the file, so wasn't sure. Fixed.